src/status_im/extensions/core.cljs

```diff
@@ -183,3 +186,63 @@
   [cofx url]
   (fx/merge (assoc-in cofx [:db :extension-url] url)
             (navigation/navigate-back)))
+
+(fx/defn set-input
+  [{:keys [db]} input-key value]
+  {:db (update db :extensions/manage assoc input-key {:value value})})
+
+(fx/defn fetch [cofx id]
+  (get-in cofx [:db :account/account :extensions id]))
+
+(fx/defn edit
+  [cofx id]
+  (let [{:keys [url]} (fetch cofx id)]
+    (fx/merge (set-input cofx :url (str url))
```

Comment:
I'm afraid you forgot to pul `cofx` as first argument to `fx/merge`

Reply:
`(set-input cofx :url (str url))` doesn't count? At least it works as expected.